megatron/neox_arguments/neox_args.py

```diff
@@ -125,6 +125,11 @@ class NeoXArgsModel(NeoXArgsTemplate):
     Normalization layer to use. Choose from "layernorm", "rmsnorm", "scalenorm".
     """
 
+    layernorm_fusion: bool = True
```

Comment:
Can we do the same config line addition that was done for fused rope? I suspect users will want to tune this knob.

Reply:
LGTM otherwise!